cmake/wasi-sdk-sysroot.cmake

```diff
@@ -143,7 +155,12 @@ endforeach()
 # libcxx build logic
 # =============================================================================
 
-function(define_libcxx target)
+execute_process(
+  COMMAND ${CMAKE_C_COMPILER} -dumpversion
+  OUTPUT_VARIABLE llvm_version
```

Comment:
Can this also be called something like `host_llvm_version` or `clang_llvm_version` to better distinguish from [`clang_version`](https://github.com/WebAssembly/wasi-sdk/blob/de6328701c9bead32fdbcabf8b177fc0c8c91011/CMakeLists.txt#L29)? (either that or can that variable be renamed?)

Reply:
my impression is clang_version should be retired. we can use -print-resource-dir instead.

Reply:
Can you retire it in this PR then? Otherwise this is creating a confusing state where both are available and they refer to slightly different things.

Reply:
i don't want to interfere with https://github.com/WebAssembly/wasi-sdk/pull/445.

Reply:
Now that #445 is merged could this be retired?

Reply:
see https://github.com/WebAssembly/wasi-sdk/pull/451